src/bio.c

```diff
@@ -257,7 +257,13 @@ void *bioProcessBackgroundJobs(void *arg) {
                     }
                 } else {
                     atomicSet(server.aof_bio_fsync_status,C_OK);
-                    atomicSet(server.pot_fsynced_reploff, job->fd_args.offset);
+                    // A negative offset indicates no replication is configured and we are tracking a basic
+                    // "fsync epoch" counter instead.
```

Comment:
if we don't have to introduce a function to abstract that negative repl offset things, and instead add a comment, we can do that for repl_backlog as well.
p.s. please avoid line comments (the only styling thing that is pretty consistent in redis)